core/core-spacemacs.el

```diff
@@ -102,6 +102,7 @@
   ;; bootstrap packages
   (spacemacs/load-or-install-protected-package 'dash t)
   (spacemacs/load-or-install-protected-package 's t)
+  (spacemacs/load-or-install-protected-package 'f t)
```

Comment:
Isn't it better to add this packages in the unimpaired layer? It will be used only there no? Maybe I'm missing something else also…